torch/csrc/distributed/autograd/init.cpp

```diff
@@ -22,7 +22,13 @@ PyObject* dist_autograd_init(PyObject* _unused, PyObject* noargs) {
     throw python_error();
   }
 
-  auto module = py::handle(autograd_module).cast<py::module>();
+  auto torch_C_module = THPObjectPtr(PyImport_ImportModule("torch._C"));
```

Comment:
seems like you are no longer using `autograd_module`, if so can you remove previous line?

also does this automatically move everything previously under `torch.distributed.autograd.*` to `torch._C._distributed_autograd.*` ? if so, we might want to be careful since I am not sure if `torch.distributed.autograd` is consider a public module or not, but from the naming the new `_distributed_autograd` is not

Reply:
> seems like you are no longer using `autograd_module`, if so can you remove previous line?

Actually I am not sure about that, because the code still checks the existence of the python module "torch.distributed.autograd". If not exist, it will throw an error. You may observe a similar pattern [here](https://github.com/pytorch/pytorch/blob/master/torch/csrc/autograd/init.cpp#L22) in the master branch. But maybe I should return a nullptr here, instead of throw python_error(). @malfet what should be the correct solution here?

> 
> also does this automatically move everything previously under `torch.distributed.autograd.*` to `torch._C._distributed_autograd.*` ? if so, we might want to be careful since I am not sure if `torch.distributed.autograd` is consider a public module or not, but from the naming the new `_distributed_autograd` is not

Yes, it moves every C++ definition that previously under "torch.distributed.autograd" to "torch._C._distributed_autograd", but I think I have fixed all related symbol references, and note that we already did similar thing for the module "torch.autograd" (i.e., move C++ exported symbols from "torch.autograd" to "torch._C._autograd" in PR##44451).